hack/build-images.sh

```diff
@@ -121,10 +127,32 @@ image openshift/hello-openshift              examples/hello-openshift
 docker build --no-cache -t openshift/deployment-example:v1 examples/deployment
 docker build --no-cache -t openshift/deployment-example:v2 -f examples/deployment/Dockerfile.v2 examples/deployment
 
+####################
```

Comment:
I was kind of expecting this to be in a `hack/build-images.sh` script in the origin-metrics and origin-logging repos.  Is that possible?